ocfweb/docs/docs/staff/procedures/accounts/renaming.md

```diff
@@ -0,0 +1,151 @@
+
+### Rename the user in Kerberos
+
+    $ kadmin rename oldname newname
+
+### Rename the user's entries in LDAP
+
+    $ kinit you/admin ldapvi uid=oldname
+
+Edit the entries that say "uid=oldname" to "uid=newname", and change the
+`homeDirectory` entry accordingly.
+
+After renaming the user in Kerberos and LDAP, flush the name cache
+by doing:
+
+    $ nscd -i passwd
+
+### Symlinks
+
+The main symlink should be `public_html` in the user's homedir which
+points to their webdir. You can delete it and either recreate it
+manually or by running `makehttp` as the new user. Make sure the
+symlinks are owned by the new user and not you or root after you're done.
+
+    $ cd ~newname && rm public_html
+    $ ln -s /services/http/users/n/newname ~newname/public_html
+
+    $ cd ~newname && rm public_html
+    $ makehttp
+
+### Print Quota
+
+Log into `pma.ocf.berkeley.edu` as `ocfprinting`. Run some SQL to
+rename the user in the quota history. For example, you might try
+
+    UPDATE `jobs` SET `user` = '<new>' WHERE `user` = '<old>';
+    UPDATE `refunds` SET `user` = '<new>' WHERE `user` = '<old>';
+
+### Move Crontabs
+
+Crontabs, among other things, are located at `supernova:/services/crontabs/`
+There are crontabs for three servers users might log into: supernova
+for staff, tsunami for regular users, and werewolves for apphosting groups.
+
+    $ sudo mv /services/crontabs/<server>/oldname \
+              /services/crontabs/<server>/newname
+
+### MySQL database
+
+Renaming databases on a live system is a bad idea, in general. Be careful
+when following what StackOverflow says to do. Hopefully the user doesn't even
+have a database.
+
+Depending on what data is in the user's database, there are a few options.
+The easiest should be to run `makemysql` on `tsunami` as the new user, and
+then delete the database of the old user.
+
+The following commands may also work, if other options are unsuitable.
+
+    $ mysqldump -u root -p -v oldname > old.sql
+    $ mysqladmin -u root -p create newname
+    $ mysql -u root -p newname < old.sql
+
+After any data migrations have taken place, run `makemysql` on `tsunami`
+anyways as the new user. It won't attempt to create a new database if one
+already exists but it will still set up grants for the new user.
+
+### Update vhosts
+
+If the account being renamed belongs to a group, it may be necessary to
+change their vhost configurations. These configurations are stored at
+`supernova:~staff/vhost/`. Look through `vhost.conf`, `vhost-app.conf`
+and `vhost-mail.conf`, editing the groups's entry if necessary to reflect
+the new name.
+
+### Documenting what you've done
+
+Log the rename after it's done by using `note`.
+
+    $ note -u newname 'renamed oldname -> newname'
+
+That should be it. Don't attempt what's written in this document verbatim,
+especially if it hasn't been updated in a while, or without being/notifying
+a site manager.
```

Comment:
Eh, DSMs I would argue would qualify for this too.
My personal opinion is that DSMs should have all of the special privileges of SMs except the ones they are explicitly not listed as having (e.g. granting root), and hold the same responsibilities.